assets/sass/components/global/_googlesitekit-help-menu.scss

```diff
@@ -0,0 +1,25 @@
+/**
+ * HelpMenu styles.
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+.googlesitekit-header .googlesitekit-user-selector .googlesitekit-help-menu__button {
+	margin-right: 0;
+}
+
+.googlesitekit-plugin .googlesitekit-help-menu .mdc-menu-surface {
+	width: auto;
+}
```

Comment:
These two might be better as utility classes rather than hidden away here.
E.g.
- `.googlesitekit-margin-right-0`
- `.googlesitekit-width-auto`

What do you think?

Reply:
Works for me! I had to tweak another, older selector to be less specific so these'd work, but it seems fine. 👍🏻